pom.xml

```diff
@@ -159,7 +159,7 @@
         <httpcore.version>4.4.16</httpcore.version>
         <hudi.version>0.15.0</hudi.version>
         <hudi.artifact>hudi-spark${spark.binary.version}-bundle_${scala.binary.version}</hudi.artifact>
-        <iceberg.version>1.6.1</iceberg.version>
+        <iceberg.version>1.7.0</iceberg.version>
```

Comment:
Currently, Kyuubi supports developing under Java 8/11/17, we should use 1.6.1 by default to avoid breaking devs who still using Java 8, and we can override `iceberg.version` in other `java-*` profiles

Reply:
It wont break the compatibility and development experience, as the `java-8` is auto-activated when running with Java 8 binging the Iceberg 1.6.x for them.